spec/system/admin/reports/enterprise_summary_fees/enterprise_summary_fee_with_tax_report_by_order.rb

```diff
@@ -0,0 +1,268 @@
+  #   1 order cycle the has:
+  #     - coordinator fees price 20
+  #     - incoming exchange fees 15
+  #     - outgoing exchange fees 10
+  #   tax
+  #       country: 2.5%
+  #       state: 1.5%
+
+  let!(:table_header){
+    [
+      "Distributor",
+      "Order Cycle",
+      "Order Number",
+      "Name",
+      "Type",
+      "Owner",
+      "Tax Category",
+      "Tax Rate Name",
+      "Tax Rate",
+      "Total excl. tax ($)",
+      "Tax",
+      "Total incl. tax ($)",
+      "First Name",
+      "Last Name",
+      "Customer Code",
+      "Customer Email"
+    ].join(" ").upcase
+  }
+
+  let!(:state_zone){ create(:zone_with_state_member) }
+  let!(:country_zone){ create(:zone_with_member) }
+  let!(:tax_category){ create(:tax_category) }
+  let!(:state_tax_rate){ create(:tax_rate, zone: state_zone, tax_category: tax_category) }
+  let!(:country_tax_rate){ create(:tax_rate, zone: country_zone, tax_category: tax_category) }
+  let!(:ship_address){ create(:ship_address) }
+
+  let!(:variant){ create(:variant) }
+  let!(:product){ variant.product }
+  let!(:distributor){ create(:distributor_enterprise_with_tax) }
+  let!(:supplier){ create(:supplier_enterprise) }
+  let!(:payment_method){ create(:payment_method, :flat_rate) }
+  let!(:shipping_method){ create(:shipping_method, :flat_rate) }
+
+  let!(:order){ create(:order_with_distributor, distributor: distributor) }
+  let!(:order_cycle){
+    create(:simple_order_cycle, suppliers: [supplier], distributors: [distributor],
+                                variants: [variant])
+  }
+
+  let(:admin){ create(:admin_user) }
+
+  let!(:coordinator_fees){
+    create(:enterprise_fee, :flat_rate, enterprise: distributor, amount: 20)
+  }
+  let!(:supplier_fees){ create(:enterprise_fee, :flat_rate, enterprise: supplier, amount: 15) }
+  let!(:distributor_fee){ create(:enterprise_fee, :flat_rate, enterprise: distributor, amount: 10) }
+
+  let!(:customer_first_name){ "Customer First Name" }
+  let!(:customer_last_name){ "Customer Last Name" }
+  let!(:customer_code){ "CUS_CODE" }
+  let!(:customer_email){ "customer1@example.com" }
+  let!(:order_number){ "ORDER_NUMBER_1" }
+
+  before do
+    state_tax_rate.update!({ name: 'State', amount: 0.015 })
+    country_tax_rate.update!({ name: 'Country', amount: 0.025 })
+    tax_category.update!({ name: 'tax_category' })
+
+    coordinator_fees.update!({
+                               name: 'Adminstration',
+                               fee_type: 'admin',
+                               tax_category: tax_category
+                             })
+    supplier_fees.update!({
+                            name: 'Transport',
+                            fee_type: 'transport',
+                            tax_category: tax_category
+                          })
+    distributor_fee.update!({
+                              name: 'Packing',
+                              fee_type: 'packing',
+                              tax_category: tax_category
```

Comment:
A little out of scope but good to keep in mind: if we change the `distributor_fee` to inherit the tax category from the product:

```
    distributor_fee.update!({
                              name: 'Packing',
                              fee_type: 'packing',
                              inherits_tax_category: true
                            })
```

we see that no taxes are charged. To my understanding this reproduces [this issue](https://github.com/openfoodfoundation/openfoodnetwork/issues/10348). It could be interesting to setup a pending example to reproduce it (I can do that in a separate PR)